ProjectSetup/Contracts/V1/UserRoles.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ProjectSetup.Contracts
```

Comment:
Move this file to the Domain part. Use contracts for a contract with the external world (Endpoints contract)